### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by mohamed82008** [JuliaStats/PDMats.jl#91]
Hi!

It would be nice to have a new release for PDMats. Thank you.

**Comment by ararslan** [comment]
I'll tag a new release once #92 is merged.

**Comment by ararslan** [comment]
@JuliaRegistrator `register()`

**Comment by mohamed82008** [comment]
Cool thanks :)
